src/mappings/primitives.ts

```diff
@@ -28,8 +33,107 @@ import {
   stringify,
   trackUnprocessed,
   unprocessedEventHandler,
+  getBalanceId,
 } from "./utils";
 
+export async function handleGenesis(block: CosmosBlock): Promise<void> {
+  const genesis: Genesis = require('../../genesis.json');
+
+  // IMPORTANT: Return early if this is not the genesis initial height as this is called on for block indexed!
```

Comment:
Apologies for injecting a typo. :sweat_smile: 

```suggestion
  // IMPORTANT: Return early if this is not the genesis initial height as this is called for block indexed!
```